test/common/index.js

```diff
@@ -315,7 +315,21 @@ const knownGlobals = new Set([
  'localStorage',
  'sessionStorage',
 ].forEach((i) => {
-  if (globalThis[i] !== undefined) {
+  if (i === 'localStorage') {
+    const getLocalStorage = Object.defineProperty(() => {}, 'name', {
+      value: 'get localStorage',
+    });
```

Comment:
This feels a bit hacky, but I wasn’t able to find a better solution. Happy to hear any suggestions.

Reply:
@danielmbrasil sorry for the long wait. Why is this needed?

Reply:
The path can point to a file in the fixture. With this, we ensure it exists. Since this test does not modify it I think it would be acceptable